cordova-lib/src/cordova/platform.js

```diff
@@ -161,8 +161,17 @@ function addHelper(cmd, hooksRunner, projectRoot, targets, opts) {
                     }
                     return superspawn.spawn(bin, args, copts);
                 }).then(function() {
-                    copy_cordova_js(projectRoot, platform);
-                    copy_cordovajs_src(projectRoot, platform, platDetails.libDir);
+                    var platform_www = path.join(projectRoot, 'platforms', platform, 'platform_www');
+
+                    // only want to copy cordova_js once, when the platform is added
+                    if (!fs.existsSync(path.join(platform_www, 'cordova.js'))) {
```

Comment:
You don't need this check. These functions only get run once when the platform gets added.